src/features/swap/index.tsx

```diff
@@ -127,7 +137,7 @@ const SwapWidget = ({ sell }: Params) => {
         event: CowEvents.ON_CHANGE_TRADE_PARAMS,
         handler: (newTradeParams) => {
           const { orderType: tradeType } = newTradeParams
-          dispatch(setSwapParams({ tradeType }))
+          dispatch(setSwapParams({ tradeType: tradeType.toLowerCase() }))
```

Comment:
how about we automatically do this conversion in the reducer and not here. I'm sure that if we have to dispatch this somewhere else we'll forget to lowercase it.